docker-compose.override.yml.example

```diff
@@ -0,0 +1,8 @@
+version: '3'
+
+services:
+
+  main-app:
+    env_file:
+      - ./app/local.env
+      - ./app/.env
```

Comment:
I'm curious if we want to name the override env file something like `override.env` to make it clear on the behavior.